testing/src/main/java/org/oppia/android/testing/TextInputAction.kt

```diff
@@ -0,0 +1,47 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import android.view.View
+import androidx.annotation.StringRes
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import javax.inject.Inject
+
+/**
+ * Action for accessing HasErrorTest and HasNoErrorTest Functions.
+ *
+ * This is needed because Robolectric doesn't seem to properly input digits for text views using
+ * 'android:digits' or other filters. See https://github.com/robolectric/robolectric/issues/5110
+ * for specifics.
+ */
+class TextInputAction @Inject constructor(
+  val context: Context
+) {
+  // TODO(#1720): Move this to a companion object & use a test-only singleton injector to retrieve
+  fun hasErrorText(@StringRes expectedErrorTextId: Int): Matcher<View> {
```

Comment:
adding kdoc for these functions as well. 
use this reference - https://developer.android.com/kotlin/style-guide#block_tags

Reply:
Function hasErrorTest which is used
in org.oppia.android.app.settings.profile and in org.oppia.android.app.profile
So instead of defining function in all the place can be accessed from here.
**For Function Describtion**

Reply:
```kotlin
/**
   * @param expectedErrorTextId is the string id which needs to match.
   * 
   * @returns a [Matcher] that matches the specified string [expectedErrorTextId] with the view's error text.
   */
```

Reply:
ok added it